resources/page/permalinks.go

```diff
@@ -305,10 +315,29 @@ func (l PermalinkExpander) pageToPermalinkSection(p Page, _ string) (string, err
 	return p.Section(), nil
 }
 
+// pageToPermalinkSectionSlug returns the URL-safe form of the first section's slug or title
+func (l PermalinkExpander) pageToPermalinkSectionSlug(p Page, attr string) (string, error) {
+	if p.Section() == "" {
+		return "", nil
+	}
+
+	sectionPage, err := p.GetPage("/" + p.Section())
```

Comment:
p.Section() returns the first level (that method was added before we had a concept of nested section), which is, I'm guessing, not what we want here. Also, I don't see why this is needed -- isn't `p` the section you want the title from?

Reply:
Yea, OK, i read this a little fast then. But then I suggest we do `sectionPage := p.FirstSection`.